vm/qemu/qemu.go

```diff
@@ -180,7 +180,8 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 	if cfg.Count < 1 || cfg.Count > 128 {
 		return nil, fmt.Errorf("invalid config param count: %v, want [1, 128]", cfg.Count)
 	}
-	if env.Debug {
+	if env.Debug && cfg.Count > 1 {
+		log.Logf(0, "Limiting number of VMs from %v to 1 in debug mode", cfg.Count)
```

Comment:
We never start log output with Capital letter anywhere.
Sorry but my eyes will bleed every time I go over this code in future :)

Reply:
Haven't figured out that myself... Fixed inhttps://github.com/google/syzkaller/pull/832/commits/fddca18d104fb4fb651eab75c111aba9eca638f3.